plugins/version-control-systems/git/src/main/kotlin/Git.kt

```diff
@@ -200,12 +225,13 @@ class Git internal constructor() : VersionControlSystem(GitCommand) {
     private fun updateWorkingTreeWithoutSubmodules(
         workingTree: WorkingTree,
         git: Git,
-        revision: String
+        revision: String,
+        historyDepth: Int
```

Comment:
If you add docs to the below function where you change the signature, you should add docs here as well for consistency.